### PR TITLE
[INFRA] Add some documentation to workflows

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

**Review by marehr — APPROVED** [review]
ty

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Builds documentation and uploads it to https://docs.seqan.de/:
+  * Any push to master
+  * Manually on any branch
+
+Uses repository secrets for the target server and authentication.
+
+## latest_libraries.yml
+
+Will update all submodules, gbenchmark and googletest to respective master versions and runs the test suite on all
```
Comment (Irallia) [inline .github/workflows/README.md:54]:
```suggestion
Will update all submodules, benchmark and googletest to respective master versions and runs the test suite on all
```

File: .github/workflows/scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+## configure_apt.sh
+
+Allows `apt` to retry operations up to 5 times and adds two ppas needed for up-to-date compilers.
```
Comment (Irallia) [inline .github/workflows/scripts/README.md:3]: What is ppas?

**Review by Irallia — CHANGES_REQUESTED** [review]
Does gbenchmark exist, or did you mean benchmark?
And what does ppas mean?

File: .github/workflows/README.md (hunk 1 above)
Comment (eseiler) [reply .github/workflows/README.md:54]: They call their lib `gbenchmark`, so I just used that name

File: .github/workflows/scripts/README.md (hunk 2 above)
Comment (eseiler) [reply .github/workflows/scripts/README.md:3]: https://launchpad.net/ubuntu/+ppas

Comment (Irallia) [reply .github/workflows/scripts/README.md:3]: thx

**Review by Irallia — APPROVED** [review]

**Pull request merged**